x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -530,6 +544,57 @@ synchronized void startAndLoad(TrainedModelLocation modelLocation, ActionListene
             }
         }
 
+        private Consumer<String> onProcessCrashHandleRestarts(AtomicInteger startsCount) {
+            return (reason) -> {
+                if (isThisProcessOlderThan1Day()) {
+                    startsCount.set(1);
+                    logger.error(
+                        "[{}] inference process crashed due to reason [{}]. This process was started more than 24 hours ago; "
```

Comment:
The ML UI has a _notifications_ page which is much more visible to the user than the logs. In the ES code base we call _notifications_ audit messages, you can see examples of auditing important messages if you look for `SystemAuditor` in `TrainedModelAssignmentClusterService`. 

As a follow up can you please audit this message and `restarting inference process after...` so that they appear in the UI